docs/kratos/social-signin/30_apple.mdx

```diff
@@ -178,6 +178,42 @@ Follow these steps to add Apple as a social sign-in provider to your project usi
 </Tabs>
 ````
 
+## Using the Apple SDK on native apps
+
+Apple provides a more integrated UX for native apps using the
+[Apple SDK](https://developer.apple.com/documentation/sign_in_with_apple/implementing_user_authentication_with_sign_in_with_apple).
+Since this flow does not involve a browser and redirect URL, but instead issues an `id_token`, it requires a bit more work to
+integrate with Ory.
```

Comment:
Always positive! :)

```suggestion
This flow uses the native Apple SDK and does not require a browser. This results in an issued `id_token`  on the client side which is exchanged at Ory for a session token.
```

Reply:
Move the limitations and drawbacks to a section at the end.

Reply:
no i meant that the drawbacks shouldn't be listed within the highlevel description.